relayer/ops/environment.go

```diff
@@ -1,45 +1,101 @@
 package ops
 
 import (
-	"github.com/smartcontractkit/chainlink-testing-framework/config"
-	"github.com/smartcontractkit/helmenv/environment"
+	"github.com/rs/zerolog/log"
+	"github.com/smartcontractkit/chainlink-env/client"
+	"github.com/smartcontractkit/chainlink-env/environment"
 )
 
-func DefaultStarkNETEnv() *environment.Config {
-	return &environment.Config{
-		NamespacePrefix: "chainlink-starknet",
-		Charts: environment.Charts{
-			"geth": {Index: 1},
-			"starknet": {
-				Index: 1,
-				Path:  "../../relayer/ops/charts/starknet",
-				Values: map[string]interface{}{
-					"real_node": false,
+type Props struct {
+	NetworkName string   `envconfig:"network_name"`
+	HttpURLs    []string `envconfig:"http_url"`
+	WsURLs      []string `envconfig:"ws_url"`
+	Values      map[string]interface{}
+}
+
+type HelmProps struct {
+	Name   string
+	Path   string
+	Values *map[string]interface{}
+}
+
+type Chart struct {
+	HelmProps *HelmProps
+	Props     *Props
+}
```

Comment:
Why doesn't the `chainlink-env` module expose these types?

They are also duplicated there: https://github.com/smartcontractkit/chainlink-env/search?q=HelmProps

Reply:
Removed and adjusted